react/package.json

```diff
@@ -27,6 +27,7 @@
         "build": "npm run transpile && npm run build:js && npm run build:js-dev && npm run tidy-up && npm run build:py && npm run tidy-up",
         "typecheck": "tsc --noEmit",
         "format": "eslint --fix *.js *json \"src/**/*.+(ts|tsx|js|jsx|json|css)\"",
+        "formatWellLogViewer": "eslint --fix \"src/lib/components/WellLogViewer/**/*.+(ts|tsx|js|jsx|json|css)\"   ",
```

Comment:
Maybe you could create your own local script instead.

Reply:
The only way to do that I know is insert it into the package.json

Reply:
I have restored package.json